app/models/user.rb

```diff
@@ -75,9 +75,9 @@ def capitalize_name_fragments
       name
         .split
         .map do |name_fragment|
-          name_fragment[0] = name_fragment[0].capitalize
-          name_fragment
-        end
+        name_fragment[0] = name_fragment[0].capitalize
+        name_fragment
+      end
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)